connectors/spark/src/test/java/io/unitycatalog/connectors/spark/SparkIntegrationTest.java

```diff
@@ -116,6 +108,66 @@ public void testDeltaPathTable() throws IOException, ApiException {
         session.stop();
     }
 
+    @Test
+    public void testManagedParquetTable() throws ApiException, IOException, ParseException {
```

Comment:
What does this have to do with "managed" table? This is only testing whether credentials vended by UC are percolated down to the relation.options. nothing more. It could be for managed tables or external tables. Table type does not matter. So the name of this test should be testCredentialUsage.

Reply:
IIUC only managed table can have credentials? BTW I really want to test the real `CREATE TABLE` statement but UC doesn't support it today. This test has to use hacky way to create managed tables in the UC server.